meshnet/server.go

```diff
@@ -440,6 +438,8 @@ func (s *Server) Invite(
 	ctx context.Context,
 	req *pb.InviteRequest,
 ) (*pb.InviteResponse, error) {
+	s.daemonEvents.Service.SendInvitation.Publish(nil)
```

Comment:
Shouldn't the event be sent only when it is successful?

Reply:
According to the documentation, in mobiles apps it is send when user clicks the button. So I assume it should be send regardless of whether action was successful or not.